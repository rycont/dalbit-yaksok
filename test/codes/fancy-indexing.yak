메뉴판: ["떡볶이", "마늘", "쑥", "자린고비", "엉겅퀴", "국화"]
메뉴판[[2, 4, 6]] 보여주기