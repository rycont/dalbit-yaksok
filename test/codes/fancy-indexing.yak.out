[마늘, 자린고비, 국화]
